posts/zh-Hans/2022-12-06-instant-on-beta.adoc

```diff
@@ -0,0 +1,1178 @@
+---
+layout: post
+title: 针对云本机 Java 应用程序的 Liberty InstantOn 启动
+# Do NOT change the categories section
+categories: blog
+author_picture: https://avatars3.githubusercontent.com/tjwatson
+author_github: https://github.com/tjwatson
+seo-title: 针对云本机 Java 应用程序的 Liberty InstantOn 启动 - OpenLiberty.io
+seo-description: Open Liberty 22.0.0.11-beta 为您带来了一个令人振奋的新功能 InstantOn ，为 Jakarta EE 和 MicroProfile 应用程序提供了令人难以置信的快速启动时间。
+blog_description: "Open Liberty 22.0.0.11-beta 为您带来了一个令人振奋的新功能 InstantOn ，为 Jakarta EE 和 MicroProfile 应用程序提供了令人难以置信的快速启动时间。"
+open-graph-image: https://openliberty.io/img/blog/instantonperf.png
+blog-available-in-languages:
+- lang: ja
+  path: /ja/blog/2022/11/01/instant-on-beta.html
+- lang: en
+  path: /blog/2022/09/29/instant-on-beta.html
+additional_authors:
+- name: Joshua Dettinger
+  github: https://github.com/dettmoney
+  image: https://avatars0.githubusercontent.com/dettmoney
+---
+= 针对云本机 Java 应用程序的 Liberty InstantOn 启动
+Thomas Watson <https://github.com/tjwatson>
+:imagesdir: /
+:url-prefix:
+:url-about: /
+
+翻译：王峰 (亚信安慧)
+
+你想让云原生Java应用在几毫秒内启动，而不影响吞吐量、内存、开发和生产等价或Java语言新特性吗？Open Liberty 22.0.0.11-beta带来了InstantOn，一个为MicroProfile和Jakarta EE应用程序提供了快速启动的新功能。
+
+在无服务器环境中，应用程序的启动时间很重要。当应用程序不被使用时，InstantOn应用程序实例可以收缩到零。当没有持续的请求时，通过减少应用实例的总体数量，收缩到零有助于降低整体云成本。当应用程序的活动增加时，新的实例可以快速启动，不会给用户体验带来高延迟。
+
+== Open Liberty是如何实现InstantOn的呢？
+
+为了实现InstantOn，Open Liberty使用了link:https://www.eclipse.org/openj9/[OpenJ9] JVM的新特性和一种叫做Checkpoint/Restore In Userspace link:https://criu.org/[CRIU]的Linux技术，在应用进程启动时对其设置检查点。这个检查点是正在运行的应用程序进程的快照，可以被持久化，也可以快速恢复，使应用程序进程回到检查点时的状态。这个过程使Liberty实例，以及任何受保护的应用程序，可以多次恢复为应用程序的不同实例。
+
+要了解InstantOn的启动速度有多快，请跳转到链接"<<benchmark, 有多快>>"。关于这种方法优势的更多细节，参考link:https://blog.openj9.org/2022/09/26/fast-jvm-startup-with-openj9-criu-support/[OpenJ9 CRIU下快速JVM启动]博文和早期的link:/blog/2020/02/12/faster-startup-Java-applications-criu.html[Open Liberty博文]，其中我们首次讨论了CRIU对快速启动Java应用程序的能力。
+
+Open Liberty InstantOn功能提供了一个新的``checkpoint``行为，可以针对你现有的Open Liberty服务器有效，你可以很容易地使用它。可以具体指定一个阶段作为检查点，让Liberty在这个检查点启动、停止、保存这个检查点。当Liberty启动时，它将检测到保存的检查点进程，并从保存的检查点中恢复进程状态。
```

Comment:
Not sure what you meant. checkpoint was not translated anyway.

Reply:
Sorry, shouldn't have used that section as an example. I translated that one `checkpoint` back to English myself using machine translation, but since I am completely unfamiliar with the language, I didn't feel comfortable translating `checkpoint` back to English throughout the article.  Linked in the original comment is the English blog, where you can see `checkpoint` is used as a special term throughout the article, so where you see `checkpoint` in special formatting in the English blog should remain in English in the Simplified Chinese blog.

Reply:
Why did you say `checkpoint` is a special word? It was not marked with a different syntax in the orginal blog. If it is in a command line, it won't be translated. It was a proper English word in the original sentence.

Reply:
![Screenshot 2023-03-24 at 2 37 15 PM](https://user-images.githubusercontent.com/84740311/227623793-d77eed3a-7ad0-43cc-8240-2101d6ee97d6.png)

In this photo of a paragraph from the English blog, the word `checkpoint` is specially formatted three times.  The formatting is not very different from the other text so it can be easy to miss, and I'll talk with the site UI team to see if we can improve that styling.

Reply:
done